superset/assets/utils/common.js

```diff
@@ -39,8 +39,19 @@ export function getParamFromQuery(query, param) {
   return null;
 }
 
-export function getLink(baseUrl, params) {
-  return baseUrl + '?' + params.join('&');
+export function getQueryLink(query, callback) {
```

Comment:
i agree with @vera-liu here. passing in the object is cleaner in my eyes as well. the objects being passed in are different in these 2 cases.

https://github.com/airbnb/superset/pull/1951/files#diff-080302e539cb3e2903e1456909802fddR45
https://github.com/airbnb/superset/pull/1951/files#diff-3917b2fc6ddaa9b01e9cea7d5518312aR12